xcb-proto.rb

```diff
@@ -25,6 +25,7 @@ def install
       --prefix=#{prefix}
       --sysconfdir=#{etc}
       --localstatedir=#{var}
+      --disable-silent-rules
```

Comment:
Yes, this formula takes `--disable-silent-rules` but not `--disable-dependency-tracking`.